samples/Meeting/src/app/utils/utils.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { GUID_FOR_INITIAL_TOPIC_NAME } from './constants';
+
+declare let __BUILDTIME__: string; // Injected by webpack
+export const getBuildTime = (): string => __BUILDTIME__;
+
+export function getChatSDKVersion(): string {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  return require('../../../package.json')['dependencies']['@azure/communication-chat'];
```

Comment:
this should be set by webpack and not grabbed from a local package.json that won't exist when the app is deployed